.github/workflows/ci.yml

```diff
@@ -0,0 +1,25 @@
+name: Test
+
+on:
+  pull_request:
+    types:
+      - edited
+      - opened
+      - reopened
+      - synchronize
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    name: Erlang/OTP ${{matrix.otp}}
+    strategy:
+      matrix:
+        otp: ['24.3', '25.3']
+        rebar3: ['3.20']
+    steps:
+      - uses: actions/checkout@v3
+      - uses: erlef/setup-beam@v1
+        with:
+          otp-version: ${{matrix.otp}}
+          rebar3-version: ${{matrix.rebar3}}
+      - run: rebar3 do eunit, ct --cover, cover
```

Comment:
Are there any additional commands we should consider running? Separately, would you prefer `eunit` and `cover` be pulled into separate steps?

Reply:
An example draft PR on my fork with the steps broken out: https://github.com/doomspork/luerl/pull/3

<img width="865" alt="image" src="https://user-images.githubusercontent.com/73386/223766942-870594e3-2dc2-481e-9861-f27ca7c30265.png">